### ISSUE TITLE
media query 

### ISSUE THREAD

**Issue by naremanhilles** [ca-g7/AMAI#51]
https://github.com/ca-g7/AMAI/blob/964a341fb1c8deb85b647d120ea3457c42f22e62/CSS/style.css#L68

it will be more better if add all media query at the bottom of file.

**Comment by Amoodaa** [comment]
yes they that happened but i had to undo everything so we can merge without conflicts
